### ISSUE TITLE
Some issues when compiling with MSVC

### ISSUE THREAD

**Issue by MellowNight** [trailofbits/maat#163]
I'm wondering if I configured anything incorrectly, these are my CMake options: 

![image](https://user-images.githubusercontent.com/66788741/211161867-a3ad8ecf-65f8-4544-bf35-3fe87ba127ce.png)
![image](https://user-images.githubusercontent.com/66788741/211161878-fc8eb472-a897-412b-b835-e9d17db636d9.png)
![image](https://user-images.githubusercontent.com/66788741/211161889-9b80dcb2-0008-490c-9ac2-9e925fac50ef.png)

I'm not sure if sleigh.exe is the right sleigh compiler to use. 

1. The usage of alternative keywords like "and," "or," etc. is not supported by MSVC. 

![image](https://user-images.githubusercontent.com/66788741/211162623-d14b0e49-3338-437b-9f03-d046f72428ad.png)

2. The always_inline attribute is not supported by MSVC either.
![image](https://user-images.githubusercontent.com/66788741/211162755-cad31047-6b9c-4f8c-aaf9-2837d9b636f6.png)

There are probably more issues than this, which I'll post in this thread in the future.

**Comment by ekilmer** [comment]
Windows support is still experimental, so there are definitely things that need to be fixed. There is a PR to track some of the changes required to get it compiled with MSVC here https://github.com/trailofbits/maat/pull/97

**Comment by mellwonight** [comment]
```

CMake Error at /home/mellownight/Downloads/cmake/share/cmake-3.25/Modules/FindPackageHandleStandardArgs.cmake:230 (message):
  Could NOT find Python3 (missing: Python3_INCLUDE_DIRS Development.Module)
  (found version "3.11.1")
Call Stack (most recent call first):
  /home/mellownight/Downloads/cmake/share/cmake-3.25/Modules/FindPackageHandleStandardArgs.cmake:600 (_FPHSA_FAILURE_MESSAGE)
  /home/mellownight/Downloads/cmake/share/cmake-3.25/Modules/FindPython/Support.cmake:3245 (find_package_handle_standard_args)
  /home/mellownight/Downloads/cmake/share/cmake-3.25/Modules/FindPython3.cmake:489 (include)
  bindings/bindings.cmake:7 (find_package)
  CMakeLists.txt:202 (include)


Configuring incomplete, errors occurred!
```

**Comment by mellwonight** [comment]
> @ekilmer I switched to linux, and cmake can't find python for some reasons.
> 
> ```
> 
> CMake Error at /home/mellownight/Downloads/cmake/share/cmake-3.25/Modules/FindPackageHandleStandardArgs.cmake:230 (message):
>   Could NOT find Python3 (missing: Python3_INCLUDE_DIRS Development.Module)
>   (found version "3.11.1")
> Call Stack (most recent call first):
>   /home/mellownight/Downloads/cmake/share/cmake-3.25/Modules/FindPackageHandleStandardArgs.cmake:600 (_FPHSA_FAILURE_MESSAGE)
>   /home/mellownight/Downloads/cmake/share/cmake-3.25/Modules/FindPython/Support.cmake:3245 (find_package_handle_standard_args)
>   /home/mellownight/Downloads/cmake/share/cmake-3.25/Modules/FindPython3.cmake:489 (include)
>   bindings/bindings.cmake:7 (find_package)
>   CMakeLists.txt:202 (include)
> 
> 
> Configuring incomplete, errors occurred!
> ```

I fixed this by installing python 3.11 alongside my existing 3.10 installation

**Comment by MellowNight** [comment]
```
[ 27%] Building CXX object CMakeFiles/maat_maat.dir/src/engine/engine.cpp.o
/home/mellownight/repos/other/maat/src/engine/engine.cpp: In member function ‘void maat::MaatEngine::load(const std::string&, maat::loader::Format, maat::addr_t, const std::vector<maat::loader::CmdlineArg>&, const maat::loader::environ_t&, const std::unordered_map<std::__cxx11::basic_string<char>, std::__cxx11::basic_string<char> >&, const std::__cxx11::list<std::__cxx11::basic_string<char> >&, const std::__cxx11::list<std::__cxx11::basic_string<char> >&, bool)’:
/home/mellownight/repos/other/maat/src/engine/engine.cpp:1454:17: error: ‘LoaderEVM’ is not a member of ‘maat::loader’; did you mean ‘Loader’?
 1454 |         loader::LoaderEVM().load(this, binary, args, envp);
      |                 ^~~~~~~~~
      |                 Loader
gmake[2]: *** [CMakeFiles/maat_maat.dir/build.make:146: CMakeFiles/maat_maat.dir/src/engine/engine.cpp.o] Error 1
gmake[1]: *** [CMakeFiles/Makefile2:326: CMakeFiles/maat_maat.dir/all] Error 2
gmake: *** [Makefile:156: all] Error 2
                                              
```
